CAM SAML configuration updated (#2685)
CenturyLinkCloud/PublicKB: Cloud Application Manager/Administering Your Organization/user-authentication.md

   * **Single Sign On URL.** Service Provider endpoint that will be used to interchange messages during authentication. This value can be found on AssertionConsumerService XML element on SP metadata file.
   * **Service Provider Entity ID.** This value must match with the entityID property on SP metadata file.

>    * **Service Provider X.509 Cert.** This value could be used to enable signing or encryption (as well as verifying signatures and decryption).

Author: Done

> 5. Finally it's needed to download the **IdP Metadata** file and upload on **Settings** page in the [Management site](https://account.cam.ctl.io/#/settings).

Author: Done

Take into consideration that currently this is our valid **Attribute mapping list**:

* **EMAIL**: 'email', 'Email', 'User.Email'.
* **FIRST NAME**: 'firstname', 'FirstName', 'User.FirstName'.
* **LAST NAME**: 'lastname', 'LastName', 'User.LastName'.
* **GROUPS** (optional): 'Groups'

### Working with SAML Groups

> SAML group members get automatic access to organizations, costcenters and team workspaces in Cloud Application Manager when you enable sharing with those groups. Through the Cloud Application Manager web or API interface, you can directly add SAML groups as members of a workspace instead of searching and adding them one by one.

Reviewer: ```suggestion
When SAML is being used in your organization and you are retrieving SAML groups information, you can add SAML groups as members of a workspace, cost-center or administrators of your organization through the Cloud Application Manager web or API interface, instead of searching and adding them one by one. All SAML group members will then get automatic access to the corresponding team workspace, cost-center or organization, and thus, to all resources pertaining to that scope.
```

Author: Done

![Adding a SAML group as Workspace member](../../images/cloud-application-manager/management/settings-auth-saml-group.png)

> This gives your developers, operations engineers, or IT admins access to the same deployment assets to do their part in automating with necessary access levels.

Reviewer: ```suggestion
This gives your developers, operations engineers, or IT admins access to the same deployment assets to do their part in automating with the necessary access levels.
```

Author: Done

LDAP group members get automatic access to team workspaces in Cloud Application Manager when you enable sharing with those groups. Through the Cloud Application Manager web or API interface, you can directly add LDAP groups as members of a workspace instead of searching and adding them one by one.

> ![Adding a LDAP group as Workspace member](../../images/cloud-application-manager/management/settings-auth-ldap-group.png)

Reviewer: ```suggestion
![Adding an LDAP group as Workspace member](../../images/cloud-application-manager/management/settings-auth-ldap-group.png)
```

Author: Done
